dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,462 @@
+
+        final String layout = String.format(jsonDrawBody, "");
+        final String testBody = String.format(body, "");
+        final Host host = new SiteDataGen().nextPersisted();
+
+        checkTemplateLayout(layout);
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(testBody)
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05380ChangeContainerPathToAbsolute task05380ChangeContainerPathToAbsolute =
+                new Task05380ChangeContainerPathToAbsolute();
+
+        task05380ChangeContainerPathToAbsolute.executeUpgrade();
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task05380ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with not filecontainer
+     * Should: Should not change
+     */
+    @Test
+    public void whenTemplateLayoutHasNotFileCOntainer() throws IOException, DotDataException, DotSecurityException {
+        final Container container = new ContainerDataGen().nextPersisted();
+
+        final String layout = String.format(jsonDrawBody, container.getIdentifier())
+                .replaceAll("/application/containers/default/", "");
+
+        final String testBody = String.format(body, "");
+        final Host host = new SiteDataGen().nextPersisted();
+
+        checkTemplateLayout(layout);
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(testBody)
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05380ChangeContainerPathToAbsolute task05380ChangeContainerPathToAbsolute =
+                new Task05380ChangeContainerPathToAbsolute();
+
+        task05380ChangeContainerPathToAbsolute.executeUpgrade();
+
+        final ArrayList templates = getTemplateFromDataBase(template);
+
+        final HashMap templateMap = (HashMap) templates.get(0);
+        assertEquals(1, templates.size());
+
+        final String drawedBody = templateMap.get("drawed_body").toString();
+        assertEquals(layout, drawedBody);
+    }
+
+    /**
+     * Method to Test: {@link Task05380ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with relative path container in the drawed_body fields and the body field is null
+     * Should: Should turn it into a Absolute Path, using the template's host
+     */
+    @Test
+    public void whenTemplateLayoutHasRelativePathButBodyIsNullShouldTurnIntoAAbsolutePath() throws IOException, DotDataException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)